lib/sms_autofill.dart

```diff
@@ -182,7 +183,7 @@ class _PhoneFieldHintState extends State<PhoneFieldHint> {
     _controller = widget.controller ?? widget.child?.controller ?? TextEditingController(text: '');
     _focusNode = widget.focusNode ?? widget.child?.focusNode ?? FocusNode();
     _focusNode.addListener(() async {
-      if (_focusNode.hasFocus && !_hintShown) {
+      if (_focusNode.hasFocus && _hintShown == false) {
```

Comment:
that's the same ^^ let's keep `!_hintShown` it can't be null anywhere

Reply:
OK